lib/Topup.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * Class Topup
+ *
+ * @property string $id
+ * @property string $object
+ * @property int $amount
+ * @property int $arrival_date
+ * @property int $created
+ * @property string $currency
+ * @property string $description
+ * @property string $failure_code
+ * @property string $failure_message
+ * @property bool $livemode
+ * @property mixed $source
+ * @property string $statement_descriptor
+ * @property string $status
+ * @property string $balance_transaction
```

Comment:
Mind alphabetizing this list (although keep `$id` and `$object` on top) and confirming that you went over it and looks correct? It looks okay to me, but I happened to notice that `$description` is still marked as undocumented so maybe we don't intend to expose that right away? Thanks!

Reply:
Hmm, I've updated the properties to reflect what should be exposed. I'm not sure I understand what you mean by `$description` being undocumented? All the fields listed now are public and should be known to users of the client.

Reply:
Thanks!

If you check the API resource in the server-side implementation, it still seems that `description` is marked as undocumented:

``` ruby
undocumented :description
```

Maybe just an oversight?

Reply:
Just to close the loop here, it looks like `description` is now an officially documented property.